qemu/tests/timerdevice_boot.py

```diff
@@ -128,27 +128,54 @@ def verify_guest_clock_source(session, expected):
     time_format = params["time_format"]
     timerdevice_drift_threshold = params.get("timerdevice_drift_threshold", 3)
 
-    error.context("Check the system time on guest and host", logging.info)
-    (host_time, guest_time) = utils_test.get_time(session, time_command,
-                                                  time_filter_re, time_format)
-    drift = abs(float(host_time) - float(guest_time))
-    if drift > timerdevice_drift_threshold:
-        raise error.TestFail("The guest's system time is different with"
-                             " host's. Host time: '%s', guest time:"
-                             " '%s'" % (host_time, guest_time))
-
-    get_hw_time_cmd = params.get("get_hw_time_cmd")
-    if get_hw_time_cmd:
-        error.context(
-            "Check the hardware time on guest and host", logging.info)
-        host_time = utils.system_output(get_hw_time_cmd)
-        guest_time = session.cmd(get_hw_time_cmd)
+    if params.get("rtc_base") == "2006-6-17":
+        guest_name = params["guest_name"]
+        if "rhel7" in guest_name:
```

Comment:
When rhel7 guest boot, chronyd service is started automatically to sync the time. rhel6 guest does not has this service. I don't know whether rhel8 guest has this service or not.